packages/sympy.rst

```diff
@@ -45,14 +41,14 @@ First Steps with SymPy
 Using SymPy as a calculator
 ---------------------------
 
-SymPy defines three numerical types: Real, Rational and Integer.
+SymPy defines three numerical types: ``Real``, ``Rational`` and ``Integer``.
 
 The Rational class represents a rational number as a pair of two
-Integers: the numerator and the denominator, so Rational(1,2)
-represents 1/2, Rational(5,2) 5/2 and so on::
+Integers: the numerator and the denominator, so ``Rational(1,2)``
+represents 1/2, ``Rational(5,2)`` 5/2 and so on::
 
-    >>> from sympy import *
-    >>> a = Rational(1,2)
+    >>> import sympy as sym
+    >>> a = sym.Rational(1,2)
```

Comment:
PEP8 tells us that we should have a space after the coma. Let us add it, while we are changing this file.

Reply:
993ebfb fixes the PEP8 throughout.